src/components/App/App.component.jsx

```diff
@@ -1,57 +1,14 @@
-import React, { useLayoutEffect } from 'react';
-import { BrowserRouter, Switch, Route } from 'react-router-dom';
-
-import AuthProvider from '../../providers/Auth';
-import HomePage from '../../pages/Home';
-import LoginPage from '../../pages/Login';
-import NotFound from '../../pages/NotFound';
-import SecretPage from '../../pages/Secret';
-import Private from '../Private';
-import Fortune from '../Fortune';
+import React from 'react';
+import HomePage from '../../pages/Home/Home.page';
+import Header from '../Header/Header';
```

Comment:
No big deal: having an `index.js` can help you with the imports, it's a good practice and it would be consistent with the rest of the project structure:

```javascript
// HomePage already has an index.js so the import could be
import HomePage from '../../pages/Home';

// Header has no index.js yet, if you create one then you can import the default component
import Header from '../Header';
```